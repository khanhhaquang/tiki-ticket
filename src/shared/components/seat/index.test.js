import React from 'react';
import renderer from 'react-test-renderer';
import Seat from './index';

describe('Test Seat component', () => {
	test('Snapshot seat with type + number', () => {
		const component = renderer.create(<Seat type='standard' number={1} />);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});

	test('Snapshot seat no type + number', () => {
		const component = renderer.create(<Seat />);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
