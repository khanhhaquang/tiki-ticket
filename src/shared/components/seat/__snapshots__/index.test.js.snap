// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Seat component Snapshot seat no type + number 1`] = `
<div
  className="sc-bdVaJa bCssIi seat "
>
  <span>
    0
  </span>
</div>
`;

exports[`Test Seat component Snapshot seat with type + number 1`] = `
<div
  className="sc-bdVaJa bCssIi seat standard"
>
  <span>
    1
  </span>
</div>
`;
